src/ResourceManager/Reservations/Commands.Reservations/packages.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.8" targetFramework="net452" />
```

Comment:
The clientruntime and newtonsoft references will come from resourcemanager.common.  This should just conating the dependency on your management client

Reply:
fixed